docs/API.md

```diff
@@ -579,28 +579,28 @@ __Parameters__
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`bucketName`  | _string_  |name of the bucket.   |
-|`objectName` | _string_  |name of the object.   |
+|`bucketName`  | _string_  |Name of the bucket  |
+|`objectName` | _string_  |Name of the object   |
 
 
 __Return Value__
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`objInfo`  | _ObjectInfo_  |object stat info for format listed below: |
+|`objInfo`  | _ObjectInfo_  |Object stat information |
 
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`objInfo.LastModified`  | _time.Time_  |modified time stamp. |
-|`objInfo.ETag` | _string_ |etag of the object.|
-|`objInfo.ContentType` | _string_ |Content-Type of the object.|
-|`objInfo.Size` | _int64_ |size of the object.|
+|`objInfo.LastModified`  | _time.Time_  |Time when object was last modified |
+|`objInfo.ETag` | _string_ |Etag of the object|
+|`objInfo.ContentType` | _string_ |Content-Type of the object|
```

Comment:
"Content type of the object."  - we don't need to write the http.Header canonical name here. Just like how we don't say "Content-Length of the object" instead we say "Size of the object".